kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/slot-updates.k

```diff
@@ -1,6 +1,6 @@
 requires "word.md"
 
-module SLOT-UPDATES [symbolic]
+module SLOT-UPDATES
```

Comment:
Removing `[symbolic]` so that the LLVM backend gets access to the helper functions.

Reply:
@ehildenb @anvacaru Perhaps there is a better way of doing this - what I would really like is to keep these modules `[symbolic]`, but just remove the auxiliary functions and put them in some module with other auxiliary functions (or a new module) that would be included with the lemmas. Is there such a file already?

Reply:
I think that could work, and they are about words, really. I've moved them, let's see.